tools/probers/workflow/workflow.go

```diff
@@ -0,0 +1,78 @@
+// workflow is a prober that runs a very basic workflow in order to test remote
+// runner health (used for remote bazel as well)
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/grpc_client"
+	"github.com/buildbuddy-io/buildbuddy/server/util/log"
+	"google.golang.org/grpc/metadata"
+
+	apipb "github.com/buildbuddy-io/buildbuddy/proto/api/v1"
+)
+
+var (
+	target = flag.String("target", "grpcs://remote.buildbuddy.io", "Buildbuddy app grpc target")
+	apiKey = flag.String("api_key", "", "The API key used to authenticate the remote run.")
+)
+
+func main() {
+	flag.Parse()
+
+	ctx := context.Background()
+	if *apiKey != "" {
+		ctx = metadata.AppendToOutgoingContext(ctx, "x-buildbuddy-api-key", *apiKey)
+	}
+	conn, err := grpc_client.DialSimple(*target)
+	if err != nil {
+		log.Fatalf("Error dialing BB target: %s", err)
+	}
+	bbClient := apipb.NewApiServiceClient(conn)
+
+	executeRes, err := bbClient.ExecuteWorkflow(ctx, &apipb.ExecuteWorkflowRequest{
+		RepoUrl:     "https://github.com/buildbuddy-io/buildbuddy",
+		Branch:      "master",
```

Comment:
since the prober repo is accessible I think this would work

```suggestion
		RepoUrl:     "https://github.com/buildbuddy-io/probers",
		Branch:      "main",
```